pages/cart-management/cart-management.css

```diff
@@ -0,0 +1,160 @@
+@import url(/main.css);
+
+
+:root {
+  --light-grey: #9ca3af;
+  --button-hover: #be185d;
+  --text-green: #16a34a;
+  --box-shadow: 0px 0px 5px 5px var(--light-grey);
+  --border-default: 1px solid;
+}
+
```

Comment:
Just a suggestion, if we can shift `:root` to main.css then you can use it everywhere.